### PR TITLE
Homework 3

### PR DESCRIPTION
**Pull request by sofa-on-the-sofa**

### CODE REVIEW

File: homework_3/js/hw_3.js [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+  console.log('none'); break;
+  case 'inline':
+  console.log('inline'); break;
+  default:
+  console.log('other');
+}
+
+//2. Из задач по условному оператору if else выполнить задачи 1, 2 и 3 в виде тернарного оператор
+//2.1 Если переменная равна “hidden”, присвоить ей значение “visible”, иначе - “hidden”
+a === 'hidden'
```
Comment (moroz112) [inline homework_3/js/hw_3.js:28]: It's good. But more common way is `a = a ==='hidden' ? 'visible' : 'hidden'`

File: homework_3/js/hw_3.js [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+//Циклы
+
+/* 1. На основе строки “i am in the easycode” сделать новую строку где первые буквы каждого слова
+будут в верхнем регистре. Использовать for или while. */
+
+let myString = 'i am in the easycode';
+let myArray = myString.split(' ');
+
+for (let i = 0; i < myArray.length; i++) {
```
Comment (moroz112) [inline homework_3/js/hw_3.js:72]: Cool) I did the same way)

**Review by moroz112 — APPROVED** [review]
Good job. Good for you

File: homework_3/js/hw_3.js (hunk 2 above)
Comment (sofa-on-the-sofa) [reply homework_3/js/hw_3.js:72]: Cool) Thanks for the review!

**Pull request merged**